js/web/lib/wasm/session-handler.ts

```diff
@@ -312,4 +329,143 @@ export class OnnxruntimeWebAssemblySessionHandler implements SessionHandler {
   endProfiling(): void {
     // TODO: implement profiling
   }
+
+  getLoggingLevel(loggingLevel: 'verbose'|'info'|'warning'|'error'|'fatal'): number {
+    switch (loggingLevel) {
+      case 'verbose':
+        return 0;
+      case 'info':
+        return 1;
+      case 'warning':
+        return 2;
+      case 'error':
+        return 3;
+      case 'fatal':
+        return 4;
+      default:
+        return 2;
+    }
+  }
+
+  setSessionOptions(allocs: number[], options?: InferenceSession.SessionOptions): number {
```

Comment:
you can return multiple values in typescript using tuple (actually it's a array, but typescript handles the types) like this:
```
function a(): [ number, string ] {
    return [ 123, "456" ];
}

const [ b, c ] = a(); // b = 123, c = "456"
```